rlclientlib/vw_model/safe_vw.cc

```diff
@@ -11,71 +11,15 @@
 namespace reinforcement_learning {
   static const std::string SEED_TAG = "seed=";
 
-  class in_memory_buf : public io_buf
```

Comment:
Should `#include "io/io_adapter.h"`

Reply:
Technically it gets included via vw.h through safe_vw.h. I can do the explicit include as well, though.